### ISSUE TITLE
Mod causes Better Trader Remake to fail loading shop contents

### ISSUE THREAD

**Issue by Kiithnaras** [probablykory/useful-trophies#3]
**Game version** `0.217.14`
**Mod version** `1.0.2`
**r2modman profile code** `018a9407-ae76-cade-deaa-a3e6a2aff0bb`

With the mod active and fresh configs, Better Trader Remake (by DigitalRoot) fails to load its shop contents. The GUI spins infinitely and the console throws the following exception:
```
[Error  : Unity Log] NullReferenceException: Object reference not set to an instance of an object
Stack trace:
UsefulTrophies.Patches.ItemDataGetTooltip (System.String __result, ItemDrop+ItemData item) (at <3c80c9975b6f4fac8da7fd54d2e398cf>:0)
(wrapper dynamic-method) ItemDrop+ItemData.DMD<ItemDrop+ItemData::GetTooltip>(ItemDrop/ItemData,int,bool,single)
ItemDrop+ItemData.GetTooltip () (at <447b19510f604ca3807cd6cce3a7c1ef>:0)
BetterTrader.Patches.StoreGui_Patches+<SetupTraderItemList>d__17.MoveNext () (at <25b30514fa424cfebc57309d5b609b68>:0)
UnityEngine.SetupCoroutine.InvokeMoveNext (System.Collections.IEnumerator enumerator, System.IntPtr returnValueAddress) (at <80fe1a4e36fe44618284312cb721a597>:0)
UnityEngine.MonoBehaviour:StartCoroutine(IEnumerator)
BetterTrader.Patches.StoreGui_Patches:SetupTraderItemListWrapper(Trader)
BetterTrader.Patches.<SetupTradeLists>d__14:MoveNext()
UnityEngine.MonoBehaviour:StartCoroutine(IEnumerator)
BetterTrader.Patches.StoreGui_Patches:FillListPrefix(StoreGui, Trader)
StoreGui:DMD<StoreGui::FillList>(StoreGui)
StoreGui:Show(Trader)
Trader:Interact(Humanoid, Boolean, Boolean)
Player:Interact(GameObject, Boolean, Boolean)
Player:DMD<Player::Update>(Player)

[Error  : Unity Log] NullReferenceException: Object reference not set to an instance of an object
Stack trace:
UsefulTrophies.Patches.ItemDataGetTooltip (System.String __result, ItemDrop+ItemData item) (at <3c80c9975b6f4fac8da7fd54d2e398cf>:0)
(wrapper dynamic-method) ItemDrop+ItemData.DMD<ItemDrop+ItemData::GetTooltip>(ItemDrop/ItemData,int,bool,single)
ItemDrop+ItemData.GetTooltip () (at <447b19510f604ca3807cd6cce3a7c1ef>:0)
BetterTrader.Patches.StoreGui_Patches+<SetupPlayerItemList>d__18.MoveNext () (at <25b30514fa424cfebc57309d5b609b68>:0)
UnityEngine.SetupCoroutine.InvokeMoveNext (System.Collections.IEnumerator enumerator, System.IntPtr returnValueAddress) (at <80fe1a4e36fe44618284312cb721a597>:0)
UnityEngine.MonoBehaviour:StartCoroutine(IEnumerator)
BetterTrader.Patches.StoreGui_Patches:SetupPlayerItemListWrapper()
BetterTrader.Patches.<SetupTradeLists>d__14:MoveNext()
UnityEngine.MonoBehaviour:StartCoroutine(IEnumerator)
BetterTrader.Patches.StoreGui_Patches:FillListPrefix(StoreGui, Trader)
StoreGui:DMD<StoreGui::FillList>(StoreGui)
StoreGui:Show(Trader)
Trader:Interact(Humanoid, Boolean, Boolean)
Player:Interact(GameObject, Boolean, Boolean)
Player:DMD<Player::Update>(Player)
```
Using Khairex's version doesn't cause this issue even though it's busted in other ways.
Let me know if I can troubleshoot further, other mods that add items (which BTR auto-populates) don't seem to cause this issue.

**Comment by probablykory** [comment]
Thanks for the report.  I'll see if I can reproduce & fix.

**Comment by probablykory** [comment]
Released 1.0.3 on tsio.  Should be fixed.
